src/Http/Http.Extensions/gen/RequestDelegateGenerator.cs

```diff
@@ -73,7 +74,7 @@ public void Initialize(IncrementalGeneratorInitializationContext context)
                 {
                     var handler = ({{endpoint.EmitHandlerDelegateCast()}})del;
                     EndpointFilterDelegate? filteredInvocation = null;
-
+{{endpoint.EmitJsonPreparation(baseIndent: 5)}}
```

Comment:
Switch over to code writer so we don't need to do this?

Reply:
Yeah, I loved having the literal string here because IMO it is so much easier to read for these snippets that don't have as much interpolation. But using the codewriter does make this much easier. See https://github.com/dotnet/aspnetcore/pull/46972 which I hoping to merge in first.